docs/src/modules/java/pages/developer-tools.adoc

```diff
@@ -0,0 +1,104 @@
+= Developer Tools
+
+include::ROOT:partial$include.adoc[]
+
+The Java tooling is published to Lightbend's public https://cloudsmith.io/~lightbend/repos/akkaserverless/packages/[Maven repository]. 
```

Comment:
This is not the case anymore. The maven artifacts are now published on Maven Central.